easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
`self` is the EasyBlock instance in this context, `self.cfg` is the parsed easyconfig file